docs/core/extensions/httpclient-factory.md

```diff
@@ -211,23 +211,13 @@ Keeping a single `HttpClient` instance alive for a long duration is a common pat
 
 ## Configure the `HttpMessageHandler`
 
-It may be necessary to control the configuration of the inner `HttpMessageHandler` used by a client.
+It may be necessary to control the configuration of the inner <xref:System.Net.Http.HttpMessageHandler> used by a client.
 
 An <xref:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder> is returned when adding named or typed clients. The <xref:Microsoft.Extensions.DependencyInjection.HttpClientBuilderExtensions.ConfigurePrimaryHttpMessageHandler%2A> extension method can be used to define a delegate on the `IServiceCollection`. The delegate is used to create and configure the primary `HttpMessageHandler` used by that client:
 
-```csharp
-services.AddHttpClient("Named.Client")
-    .ConfigurePrimaryHttpMessageHandler(() =>
-    {
-        return new HttpClientHandler
-        {
-            AllowAutoRedirect = false,
-            UseDefaultCredentials = true
-        };
-    });
-```
+:::code source="snippets/http/configurehandler/Program.cs" id="configurehandler":::
```

Comment:
Shouldn't this have a language identifier?

Reply:
Nope, I mean we could add it but the underlying tech knows that `.cs` extension files are a `csharp` slug.